Bang/GameEvents/CardEffects/States/HandlerState.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Bang.GameEvents.CardEffects.States
 {
-    internal abstract class HandlerState
+    public abstract class HandlerState
```

Comment:
`HandlerState` is the implementation detail. Other assemblies should not know about it

Reply:
done